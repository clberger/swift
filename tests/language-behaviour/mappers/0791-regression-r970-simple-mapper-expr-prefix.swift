type messagefile;

(messagefile t) write() { 
    app {
        echo @filename(t) stdout=@filename(t);
    }
}

string a = "0791-regression-r970";
string b = "-simple-mapper-";
string c = "expr-prefix";

messagefile outfile <simple_mapper;prefix=@strcat(a,b,c),suffix=".out">;

outfile = write();

