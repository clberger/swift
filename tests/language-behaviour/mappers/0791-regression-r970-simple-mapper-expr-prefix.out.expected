0791-regression-r970-simple-mapper-expr-prefix.out
